webpack/assets/javascripts/react_app/common/helpers.js

```diff
@@ -163,6 +163,15 @@ export const stringIsPositiveNumber = value => {
 export const getManualURL = section => foremanUrl(`/links/manual/${section}`);
 export const getWikiURL = section => foremanUrl(`/links/wiki/${section}`);
 
+/**
+ * Transform bytes to human readable size
+ */
+export const formatBytes = (bytes) => {
```

Comment:
We already have `number_to_human_size` from `number_helpers` in the codebase (see `webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/index.js`).